webapp/i18n/fr.json

```diff
@@ -2091,6 +2091,7 @@
   "search_results.usagePin4": "Pour épingler un message : Allez dans le message que vous souhaitez épingler et cliquez sur [...] > \"Épingler au canal\".",
   "setting_item_max.cancel": "Annuler",
   "setting_item_max.save": "Enregistrer",
+  "setting_item_max.saving": "Enregistrement ...",
```

Comment:
Non-English strings are usually submitted on our translation server, https://translate.mattermost.com/, so these might need to be removed. I'll double check if they can be left in though

Reply:
This doesn't need to be removed, but it will get overwritten with the next translation PR, so you don't need to worry about doing the translations in the future